src/include/connection.h

```diff
@@ -621,6 +621,7 @@ struct __wt_connection_impl {
     uint32_t evict_threads_max; /* Max eviction threads */
     uint32_t evict_threads_min; /* Min eviction threads */
     bool evict_sample_inmem;
+    bool evict_use_npos;
```

Comment:
If this is not wt_shared, why we need to use atomic_load to read it?

Reply:
Added